Cargo.toml

```diff
@@ -10,15 +10,18 @@ readme = "README.md"
 keywords = ["geo", "geospatial", "wkt"]
 
 [dependencies]
-geo-types = {version = "0.7.1", optional = true}
+geo-types = { version = "0.7.1", optional = true }
 num-traits = "0.2"
+serde = { version = "1.0", default-features = false, optional = true }
 thiserror = "1.0.23"
 
 [dev-dependencies]
 criterion = { version = "0.2" }
+serde = { version = "1.0", default-features = false, features = ["derive"] }
+serde_json = "1.0"
 
 [features]
-default = ["geo-types"]
+default = ["geo-types", "serde"]
```

Comment:
From what I've seen in the ecosystem, Serde support is usually opt-in and not included as a default feature. If I had to guess, I'd say more often then not, users won't need Serde support when using this crate. Would love to hear from others

Reply:
Yeah, I think you're right. It should probably not be included in `default`.

Reply:
Oops, this was definitely not intended as I used it only locally to facilitate the development. Good catch, thank you. I amended the last commit.